AssettoServer/Commands/Modules/AdminModule.cs

```diff
@@ -108,11 +110,29 @@ public void SetWeather(int weatherId)
         }
     }
 
+    [Command("cspweather")]
+    public void CspWeather()
+    {
+        List<WeatherFxType> weathers = Enum.GetValues<WeatherFxType>().ToList();
```

Comment:
really just a nitpick but you could remove the `ToList` and save one list allocation

Reply:
yeah your right, dont use c# very often (mostly c/cpp/rust) so i might miss some details like this
fixed